### ISSUE TITLE
"Files to include" and "Files to exclude" have 'search include pattern' and 'search exclude pattern' as aria labels

### ISSUE THREAD

**Issue by v-kanve** [microsoft/vscode#122839]
GitHubTags:#A11y_VSCode;#A11ySev1;#A11yTCS;#A11yMAS;#DesktopApp;#Visual Studio Code;#Label in name;#JAWS;#ScreenReader;#WCAG2.5.3;#Win32

### Environment Details:​
Application Name: Visual Studio Code
Application Version: 1.55.2 (user setup)
Date: 2021-04-13T09:35:57.887Z
Electron: 11.3.0
Chrome:87.04280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Windows_NT x64 10.0.19042
Screen Reader: JAWS
Standard Violated: WCAG2.5.3

### Repro Steps:​
1. Turn on JAWS
2. Open Visual Studio Code editor
3. Tab till "Search button" in the menu bar and press enter
4. Tab till "Files to include", "Files to exclude" controls
5. Verify whether JAWS is reading the exact label names "Files to include" ,"Files to exclude" or not

### Actual Result:
JAWS is not reading the correct label names for "files to include" and "files to exclude" controls. It is announcing as search include pattern and search exclude pattern.

### Expected Result: ​
JAWS should announce the correct label names for "Files to include" and "Files to exclude" controls. After reading the label names it should give information about the control.

### User Impact:
Screen Reader users will face difficulty while navigation if JAWS is not announcing the correct label names. They will not know the exact label name of the controls.

### Attachment:
[A11y_Visual Studio Code_Search_Label in name JAWS is not reading the correct label names for files to include and files to exclude controls. It is announcing as search include pattern.zip](https://github.com/microsoft/vscode/files/6414185/A11y_Visual.Studio.Code_Search_Label.in.name.JAWS.is.not.reading.the.correct.label.names.for.files.to.include.and.files.to.exclude.controls.It.is.announcing.as.search.include.pattern.zip)

**Comment by isidorn** [comment]
Disagree. "Search include pattern" and "Search exclude pattern" are good aria label names, we do not have to use the same as the visible labels.
@JacksonKearl forwarding to you to decide if you want to close this as designed or if you would prefer to update the aria labels
